src/commands.ts

```diff
@@ -7,8 +7,14 @@ export namespace Commands {
 
   export const JAVA_RESOLVE_MAINMETHOD = 'vscode.java.resolveMainMethod';
 
+  export const JAVA_RESOLVE_CLASSFILTERS = 'vscode.java.resolveClassFilters';
+
+  export const JAVA_UPDATE_DEBUG_SETTINGS = "vscode.java.updateDebugSettings";
```

Comment:
Let's stick to `'` for consistency. I somehow missed this before. Use `npm run lint` to find issues and `npm run fix` to fix them.
```suggestion
  export const JAVA_UPDATE_DEBUG_SETTINGS = 'vscode.java.updateDebugSettings';
```